aws/data_source_aws_rds_orderable_db_instance.go

```diff
@@ -204,6 +218,54 @@ func dataSourceAwsRdsOrderableDbInstanceRead(d *schema.ResourceData, meta interf
 				}
 			}
 
+			if v, ok := d.GetOk("supports_enhanced_monitoring"); ok {
```

Comment:
Just noting here that this logic won't work with `false` configuration values, e.g. finding instance options without enhanced monitoring in this case. This is likely the desired behavior since we do not want to lookup the boolean default of false if the argument is unspecified in the configuration (if we switched this to `d.Get()` instead), but could potentially be confusing if someone is expecting `false` to work for some reason. This is one of the very few cases `d.GetOkExists()` should actually work, but we avoid that call usually and should continue that trend because its behavior is not well understood.

It may be worth adjusting the documentation for these to say something along the lines of "enable this to ensure xxx is supported"

Reply:
Changes made!